.env.local.sample

```diff
@@ -14,3 +14,4 @@ ADMIN_EMAIL=admin@example.com
 ADMIN_PASSWORD= password
 BAMBOOHR_SUBDOMAIN=<Ask a team mate>
 BAMBOOHR_API_KEY=<Ask a team mate>
+MAPBOX_TOKEN=
```

Comment:
What is the mapbox_token used for?

Reply:
Mapbox requires the user to have a token to use its API at all. The map won't render without one.

Reply:
Or maybe you're asking why I added part of that assignment line to the env - which is because I thought all keys and tokens were referenced in that file even if the user doesn't have them explicitly in their .env (is that wrong?)

Reply:
That's right, but the intent is to use the rails API as layer to keep API keys out of the frontend. Leaking them through to the frontend isn't going to be viable. 

Is there a limit for the requests or a pricing plan for the API? How do we go about making new API keys?

Reply:
The Tokens API I used for free comes at a cap of 100 requests/min, which if exceeded simply returns a "HTTP 429 Too Many Requests" response from the API - and throttles further interaction. Anybody can create a mapbox token api key with only their email address - which is a lot better than the alternative of using the google maps API (my original implementation) where key users must sign up by providing their card number.

Reply:
Should I include some instructions in the helios README?

Reply:
:thinking: The issue is the API key with the 100 req/min limit will be available in our bundle for anyone to access. If they do that, and hit our cap, our frontend stops working. What is the API used for? Can we pass the request through the rails server so we don't expose the API key?

Reply:
I will try to pass the request through the rails server, and I intend for (like with the openWeather API key?) everyone to create their own token. I hope I'm understanding your suggestion correctly - does this solve all of the issues?

Reply:
This would solve exposing the API key issue.

Can you link to the details for the API key request limits? There's usually a monthly cap in addition to a req/min cap.

Reply:
https://docs.mapbox.com/api/accounts/tokens/#list-tokens (bottom of this page for request limits)
https://docs.mapbox.com/accounts/guides/invoices/ - the beginning of this one has additional useful information